gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/AvatarsSection.kt

```diff
@@ -37,6 +37,7 @@ internal fun AvatarsSection(
     state: AvatarsSectionUiState,
     onAvatarSelected: (AvatarUi) -> Unit,
     onAltTextSelected: (AvatarUi) -> Unit,
+    onDeleteSelected: (AvatarUi) -> Unit,
```

Comment:
💡 The number of callbacks will grow quite a bit. I had the same problem when adding "Download" option and figured we could have one callback with an extra param. Here's the commit, that you can check or even cherry-pick: https://github.com/Automattic/Gravatar-SDK-Android/commit/ceb89ace810962d769b29a0ff43903c6a702e3c9


Also, we don't have to pass the `AvatarUi` here - we only need these callbacks for the images that are uploaded to the server. The above commit tackles that as well.

Reply:
Great suggestion @AdamGrzybkowski! It looks much better with your refactor. I've cherry-picked your commit and resolved the conflicts. Thanks!